scripts/disputed_relation_overpass_query.txt

```diff
@@ -0,0 +1,111 @@
+// Contains the relations needed for a filtered_dispute package.
+// Recommended to download fresh package with each new Daylight release then
+// compare with previous package for edits or deletions that may need correction
+// or updating before using in a build.
+
+[out:xml][timeout:25];
+(
+rel(9520114); // Extent of Moroccan Claim at Melilla
```

Comment:
How brittle is this? How does someone know to add a new relation (and which one) to this list?

Reply:
That's a good question we don't have a great answer to at the moment.

In general, the answer is when NE updates, adds, or removes a relation, then this list should follow. No one else (so far) is making disputed relations in quite the same way as us so it will presumably be an active decision to create a new one on our part rather than locate an existing one.